packages/maskbook/src/plugins/ITO/UI/ClaimDialog.tsx

```diff
@@ -224,15 +231,11 @@ export function ClaimDialog(props: ClaimDialogProps) {
                     className={classes.swapLimitSlider}
                     value={Number(tokenAmount.dividedBy(maxSwapAmount).multipliedBy(100))}
                     onChange={(_, newValue) => {
-                        let tAmount = maxSwapAmount.multipliedBy((newValue as number) / 100)
-                        const swapAmount = formatBalance(tAmount.multipliedBy(ratio), claimToken.decimals)
```

Comment:
After adopt `BigNumber.dp(0)` in https://github.com/DimensionDev/Maskbook/pull/2365, remove the incorrect `formatBalance()` which should not be used in the process of token conversion calculating. Then the slider undraggable bug fixed